src/utils/assets.js

```diff
@@ -469,10 +463,15 @@ export const getAssetOptionSortPriority = ({ symbol, balance, imageUrl }: AssetO
   return 0;
 };
 
-export const mergeAccountAssets = (assetsByAccount: AssetsByAccount): Assets => {
-  const assetsArray = Object.keys(assetsByAccount).map(accountId => assetsByAccount[accountId]);
-  return Object.assign({}, ...assetsArray);
-};
+export const getCrossChainAccountAssets = (
```

Comment:
This concept of this function looks dangerous, as assets used on multiple chain will overwrite each other, e.g. MATIC on ETH & Polygon. ❗

Reply:
Actually this is not even used, was part of previous approach and left unused, removed.